contribs/gmf/src/directives/partials/gmflayertree.html

```diff
@@ -0,0 +1,19 @@
+<span ng-if="::!layertreeCtrl.isRoot"
+    ng-model="::gmfLayertreeCtrl.getStyleGetterSetter(layertreeCtrl)"
+    ng-model-options="{getterSetter: true}">
+  <img ng-src="{{::layertreeCtrl.node.icon}}"></img>
+  <input type="checkbox"
```

Comment:
You're right, that will be icons.
I'll move the logic into ng-click/ng-class, without these gettersetter.

Thanks @fgravin